spec/mainadditions.html

```diff
@@ -16,22 +17,17 @@ <h1><a href="https://tc39.es/ecma262/#sec-ecmascript-language-types-string-type"
+        The definitions include minor editorial changes from ECMA-402 to address feedback from the ECMA-262 editors.
       </p>
     </emu-note>
 
     <p>[...]</p>
 
     <ins class="block">
-      <p>
-        The <dfn>ASCII-uppercase</dfn> of a String value _S_ is the String value derived from _S_ by replacing each occurrence of an ASCII lowercase letter code unit (0x0061 through 0x007A, inclusive) with the corresponding ASCII uppercase letter code unit (0x0041 through 0x005A, inclusive) while preserving all other code units. The ASCII-uppercase of a sequence of Unicode code points _A_ is the sequence of code points derived from _A_ by replacing each occurrence of an ASCII lowercase letter code point (U+0061 through U+007A, inclusive) with the corresponding ASCII uppercase letter code point (U+0041 through U+005A, inclusive) while preserving all other code points.
-      </p>
-      <p>
-        The <dfn>ASCII-lowercase</dfn> of a String value _S_ is the String value derived from _S_ by replacing each occurrence of an ASCII uppercase letter code unit (0x0041 through 0x005A, inclusive) with the corresponding ASCII lowercase letter code unit (0x0061 through 0x007A, inclusive) while preserving all other code units. The ASCII-lowercase of a sequence of Unicode code points _A_ is the sequence of code points derived from _A_ by replacing each occurrence of an ASCII uppercase letter code point (U+0041 through U+005A, inclusive) with the corresponding ASCII lowercase letter code point (U+0061 through U+007A, inclusive) while preserving all other code points.
-      </p>
-      <p>
-        A String value _A_ is an <dfn>ASCII-case-insensitive match</dfn> for String value _B_ if the ASCII-uppercase of _A_ is exactly the same sequence of code units as the ASCII-uppercase of _B_. A sequence of Unicode code points _A_ is an ASCII-case-insensitive match for _B_ if _B_ is an ASCII-case-insensitive match for ! CodePointsToString(_A_).
-      </p>
+      <p>The <dfn>ASCII-uppercase</dfn> of a String _S_ is the String derived from _S_ by replacing each occurrence of an ASCII lowercase letter code unit (0x0061 through 0x007A, inclusive) with the corresponding ASCII uppercase letter code unit (0x0041 through 0x005A, inclusive) while preserving all other code units.</p>
+      <p>The <dfn>ASCII-lowercase</dfn> of a String _S_ is the String derived from _S_ by replacing each occurrence of an ASCII uppercase letter code unit (0x0041 through 0x005A, inclusive) with the corresponding ASCII lowercase letter code unit (0x0061 through 0x007A, inclusive) while preserving all other code units.</p>
+      <p>A String _A_ is an <dfn variants="ASCII-case-insensitive matches,ASCII-case-insensitive,ASCII-case-insensitive comparison,ASCII-case-insensitive comparisons">ASCII-case-insensitive match</dfn> for a String _B_ if the ASCII-lowercase of _A_ is the ASCII-lowercase of _B_.</p>
```

Comment:
I admit that I didn't understand the code points thing. Where does it apply in 402? 

BTW, the 262 editors asked me to take the code points text out, before they asked me to take out the entire case-insensitivity section.

Reply:
> I admit that I didn't understand the code points thing. Where does it apply in 402?

Arguably in [IsValidTimeZoneName](https://tc39.es/ecma402/#sec-isvalidtimezonename), because the current state of ECMA-402 is not clear about whether names from the IANA Time Zone Database are instances of String (i.e., [sequences of UTF-16 code units](https://tc39.es/ecma262/multipage/ecmascript-data-types-and-values.html#sec-ecmascript-language-types-string-type)) or [sequences of code points](https://tc39.es/ecma262/multipage/ecmascript-language-source-code.html#sec-codepointstostring). But I guess that's all being replaced by Temporal anyway, so there's no need to mention code points if we're clear about time zone identifiers always being String values. Suggestion withdrawn.